tools/st2ctl

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
+
+set -e
+
 LOGFILE="/tmp/st2_startup.log"
 COMPONENTS="actionrunner st2api sensor_container rules_engine mistral st2resultstracker"
 STANCONF="/etc/st2/st2.conf"
 PYTHON=`which python`
-if [ -z "$AR" ];
```

Comment:
No.  I was not clear from this script what that block was doing because $AR didn't seem to come from anywhere.